### PR TITLE
fix webhook endpoint input sizing

### PR DESCRIPTION
**Pull request by varsanojidan**
Fixes https://github.com/sourcegraph/sourcegraph/issues/45249

Small change to fix the input field from overlapping with the copy button

https://user-images.githubusercontent.com/20795805/205960218-449a5b54-e5f9-4f28-b1c3-abe024710b5b.mp4

## Test plan
Ran locally, verified no overlap

## App preview:

- [Web](https://sg-web-iv-webhook-bug.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Review by ryanslade — APPROVED** [review]
Result looks good to me, no idea if the code is idiomatic :)

File: client/web/src/site-admin/WebhookInformation.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const WebhookInformation: FC<WebhookInformationProps> = props => {
                 <tr>
                     <th className={styles.tableHeader}>Webhook endpoint</th>
                     <td className={styles.contentCell}>
-                        <CopyableText text={webhook.url} size={webhook.url.length} />
+                        <CopyableText text={webhook.url} size={webhook.url.length > 60 ? 60 : webhook.url.length} />
```
Comment (mrnugget) [inline client/web/src/site-admin/WebhookInformation.tsx:35]:
```suggestion
                        <CopyableText text={webhook.url} size={60} />
```

Why not?

**Review by mrnugget — APPROVED** [review]

File: client/web/src/site-admin/WebhookInformation.tsx (hunk 1 above)
Comment (varsanojidan) [reply client/web/src/site-admin/WebhookInformation.tsx:35]: My thought here was that in the scenario that the Webhook Endpoint length is < 60, you would have a bunch of empty space in the input between the text and the copy button, which I thought looks off.

Comment (varsanojidan) [reply client/web/src/site-admin/WebhookInformation.tsx:35]: Like this:
image

Comment (mrnugget) [reply client/web/src/site-admin/WebhookInformation.tsx:35]: I think that's fine though

Comment (varsanojidan) [reply client/web/src/site-admin/WebhookInformation.tsx:35]: To me it looked a little off, but I'm not super opinionated about it.

**Pull request merged**